CBLClient/Apps/JavaRequestHandlers/src/main/java/com/couchbase/mobiletestkit/javacommon/RequestHandler/PeerToPeerRequestHandler.java

```diff
@@ -52,6 +52,7 @@ public Replicator configure(Args args) throws Exception {
         String tlsAuthType = args.get("tls_auth_type");
         Boolean serverVerificationMode = args.get("server_verification_mode");
         Boolean tlsAuthenticator = args.get("tls_authenticator");
+        String heartbeat = args.get("heartbeat");
```

Comment:
What about other platforms ?

Reply:
also functional test for Message listener and Url Listener

Reply:
same as above

Reply:
added support for ios and dotnet.

Reply:
I believe the configure() function in PeerToPeerRequestHandler is the only place to set p2p replicator config, so it should work for message endpoint and url endpoint listeners.

Reply:
Correct, we need to use that param in the python code right